HiP-DataStore.Model/Rest/RatingArgs.cs

```diff
@@ -0,0 +1,16 @@
+﻿using System;
+using System.ComponentModel.DataAnnotations;
+
+namespace PaderbornUniversity.SILab.Hip.DataStore.Model.Rest
+{
+    public class RatingArgs
+    {
+        [Required]
+        [Range(1, 5, ErrorMessage = "Has to be in a range from 1 to 5")]
```

Comment:
Perhaps use `RatingEntityInfo.MinRateValue` and `RatingEntityInfo.MaxRateValue` here?